Library/Formula/opencoarrays.rb

```diff
@@ -0,0 +1,73 @@
+class Opencoarrays < Formula
+  desc "Coarray Fortran runtime library and toolchain, free & open source"
+  homepage "http://opencoarrays.org"
+  url "https://github.com/sourceryinstitute/opencoarrays/releases/download/1.2.0/opencoarrays-1.2.0.tar.gz"
+  sha256 "7d0c3f2dd842b32b2a24ac5fb923bc026d541d9e037205162b39a3d381d9fa83"
+
+  head "https://github.com/sourceryinstitute/opencoarrays.git"
+
+  option "without-test", "Run build time tests (requires properly configured MPICH and firewall settings)"
+
+  depends_on :fortran
+  depends_on "gcc"
+  depends_on :mpi => [:cc, :f90, "with-fortran"]
+  depends_on "cmake" => :build
+
+  fails_with :clang do
+    cause "Only compatible with GCC >= 5.1"
+  end
+
+  fails_with :llvm do
+    cause "Only compatible with GCC >= 5.1"
+  end
+
+  fails_with :gcc => 5.0 do
```

Comment:
This syntax requires a string on the right-hand side of `=>`, not a float. Also, there was never a GCC 5.0 release. You might want to consolidate this and the preceding statements based on the suggestions made in #46551.